README.md

```diff
@@ -1,12 +1,18 @@
 # trino-properties-index
 
-An index of all available properties in different versions of Trino
+An index of all available properties in different releases of Trino.
 
 ## Usage
-1. Install correct Java version (see `.java-version`) and make sure the `JAVA_HOME` environmental variable is set correctly.
-2. Run `./mvnw package` to build the project. 
-3. Run `properties/update.sh` to extract properties.
-4. Run `report.sh -s 372 -t 399` to list added and removed properties between versions 372 (source) and 399 (target), inclusive.
+Install correct Java version (see `.java-version`) and make sure the `JAVA_HOME` environmental variable is set correctly.
+Run `./mvnw package` to build the project. 
+
+### Search the property index
+
+The tool uses [sqlite](https://www.sqlite.org/) to create an in-memory database out of the Trino properties files stored
+locally in the [properties](properties) folder of the project and is able to therefore swiftly all the Trino properties
+changes between two Trino releases.
```

Comment:
Missed a word, and the sentence is too long.
```suggestion
The tool uses [sqlite](https://www.sqlite.org/) to create an in-memory database
out of the Trino properties files stored locally in the [properties](properties)
folder. Using an SQL query, it can search through all the Trino properties changes
between any two Trino releases.
```